kubernetes-model/pkg/schemagen/generate.go

```diff
@@ -30,29 +30,36 @@ type PackageDescriptor struct {
 }
 
 type schemaGenerator struct {
-	types              map[reflect.Type]*JSONObjectDescriptor
-	typeNames          map[reflect.Type]string
-	customTypeNames    map[string]string
-	packages           map[string]PackageDescriptor
-	typeMap            map[reflect.Type]reflect.Type
+	types           map[reflect.Type]*JSONObjectDescriptor
+	typeNames       map[reflect.Type]string
+	customTypeNames map[string]string
```

Comment:
are you sure about the tab spacing here?

Reply:
Umm, I'm not sure. I just used `go fmt kubernetes-model/pkg/schemagen/generate.go` for formatting. Do you think it can have some side effects?

Reply:
oh. If `go fmt` says so then its alright :)